test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -792,6 +792,22 @@ def forward(self, input):
         model = CumSum()
         self.run_test(model, x)
 
+    def test_log(self):
```

Comment:
Added this additional test point for torch.log, which was already supported but did not have a test.